### PR TITLE
[feature-fix] Contributor Visibility Logs -- Anonymous VOLs 

### PR DESCRIPTION
**Pull request by caseyrollins**
- Adds custom log language for contributor visibility changes in anonymous view only links.
- [OSF-5331]

### CODE REVIEW

**Comment by mfraezz** [comment]
Pass complete, confirmed that behavior is as expected. Sending this to RTM :mouse: 

screen shot 2016-02-12 at 14 53 19

File: website/templates/log_list.mako [hunk 1]
```diff
@@ -54,8 +54,19 @@
                                         <span class="overflow" data-bind="text: log.userFullName"></span>
                                     </span>
                                 </span>
-                                <!-- Log actions are the same as their template name -->
-                                    <span data-bind="template: {name: log.action, data: log}"></span>
+                                <!-- Handle specific language for contributor bibliographic/non-bibliographic on anonymous VOLs -->
+                                    <!-- ko if: log.action == 'made_contributor_visible' && log.anonymous -->
```
Comment (sloria) [inline website/templates/log_list.mako]: This doesn't look like the right place for this logic. Can you put this logic into `log_templates.mako` instead?

**Comment by sloria** [comment]
Pass finished; just the one suggestion/question above.

**Comment by caseyrollins** [comment]
@sloria, moved the logic into ```log_templates.mako``` :octopus:

**Comment by mfraezz** [comment]
LGTM. Build failure is from a jshint error that got into `develop`, but nothing in this should affect tests.

**Pull request merged**